src/components/dialog/Dialog.spec.js

```diff
@@ -69,6 +69,38 @@ describe('BDialog', () => {
         expect(wrapper.vm.onConfirm).toHaveBeenCalled()
     })
 
+    it('async confirm and keep Loading state', async () => {
+        wrapper.setProps({
+            onConfirm: jest.fn((confirmValue, { startLoading }) => {
+                startLoading()
+                expect(wrapper.vm.isLoading).toBeTruthy()
+                return new Promise(() => {})
```

Comment:
This line is unnecessary because `Dialog` discards the returned value from `onConfirm`.
Please see https://github.com/buefy/buefy/blob/f4c72b9d897b04c75d5a439c6de61a9ae66138dc/src/components/dialog/Dialog.vue#L212

Reply:
If I remove the return statement, the new keyword will cause an error. Should I delete that entire line? Or is there another way to fix it?

Reply:
Let's move on to another question. How can we simulate asynchronous requests?

Reply:
> If I remove the return statement, the new keyword will cause an error. Should I delete that entire line? Or is there another way to fix it?

Please remove the entire line 77. It has no effect anyway.

Reply:
done